pages/help/[page].tsx

```diff
@@ -0,0 +1,114 @@
+import addSiteLocation from '../../content/help/addSiteLocation.md';
```

Comment:
This import name can stay `addSiteLocation` even if you rename the file as I mentioned above.